easybuild/easyconfigs/p/PyTorch/PyTorch-1.12.0-foss-2022a.eb

```
@@ -0,0 +1,97 @@
+name = 'PyTorch'
+version = '1.12.0'
+
+homepage = 'https://pytorch.org/'
+description = """Tensors and Dynamic neural networks in Python with strong GPU acceleration.
+PyTorch is a deep learning framework that puts Python first."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/pytorch',
+        'repo_name': 'pytorch',
+        'tag': 'v%(version)s',
+        'recursive': True,
+    },
+}]
```

```suggestion
source_urls = ['https://github.com/%(github_account)s/%(namelower)s/releases/download/v%(version)s']
sources = ['%(namelower)s-v%(version)s.tar.gz']
```

@Flamefire got the PyTorch devs to package up the complete tree in one file for us to download instead of doing the git clone: [pytorch-v1.12.0.tar.gz](https://github.com/pytorch/pytorch/releases/download/v1.12.0/pytorch-v1.12.0.tar.gz)

---

:+1: